api/src/Service/Auth/AuthService.php

```diff
@@ -123,12 +77,7 @@ public function getUserByEmailAndPassword($email, $pass)
         return null;
     }
 
-    /**
-     * @param string $token
-     *
-     * @return User|null
-     */
-    public function getUserByToken($token)
+    public function getUserByToken($token): User|null
```

Comment:
Again should we type-hint the parameter?

Reply:
As above, leaving this change to the PHPStan tickets.